test/test_gpu_gather_ops.cpp

```diff
@@ -66,7 +74,7 @@ at::Tensor generateScatter2DIndex(
 
 } // namespace
 
-TEST_F(NVFuserTest, FusionScatter1DIndexZerosSelfTvSameShape_CUDA) {
+TEST_F(NVFuserTest, Scatter1DIndexZerosSelfTvSameShape_CUDA) {
```

Comment:
nit: It would also be good to rename `NVFuserTest` as `IndexingOpTest`.

Reply:
Done